src/palindrome.py

```diff
@@ -0,0 +1,5 @@
+def is_palindrome(checked_str):
```

Comment:
Maybe with numbers we are going to have trouble.

Reply:
There was no test for it.